ups/drp_pfs_data.table

```diff
@@ -1 +1,3 @@
 setupRequired(sconsUtils)
+
+envPrepend(PATH, ${PRODUCT_DIR}/bin)
```

Comment:
I'm not clear what this step is meant to do. Can you clarify?

Reply:
This is the standard command in an EUPS table file to include the package's `bin` directory in the `PATH` envvar.